scribe.perl

```diff
@@ -1201,7 +1225,10 @@ ($$)
   n => "<p class=anchor id=\"%2\$s\"><a href=\"#%2\$s\">⚓</a></p>\n",
   u => "<p id=%2\$s class=issue><strong>Issue:</strong> %3\$s</p>\n",
   T => "<h4 id=%2\$s>%3\$s</h4>\n",
-  t => "</section>\n\n<section>\n<h3 id=%2\$s>%3\$s</h3>\n");
+  t => "</section>\n\n<section>\n<h3 id=%2\$s>%3\$s</h3>\n",
+  slideset => "<p class=summary>Slideset: %3\$s</p>",
+  slide => "<i-slide src=\"%2\$s\"><p class=summary>[ <a href=\"%2\$s\">Slide %3\$s</a> ]</p></i-slide>"
```

Comment:
```suggestion
  slide => "<p class=summary><i-slide src=\"%2\$s\">[ <a href=\"%2\$s\">Slide %3\$s</a> ]</i-slide></p>"
```

Reply:
Adapt to i-slide defaulting to be `inline-block` per https://github.com/w3c/i-slide/pull/23#issuecomment-924788404 cc @tidoust